pkg/numatopo/update.go

```diff
@@ -81,6 +83,11 @@ func CreateOrUpdateNumatopo(client *versioned.Clientset) {
 			NumaResMap:  GetAllResAllocatableInfo(),
 			CPUDetail:   GetCpusDetail(),
 		}
+		if numaInfo.Annotations == nil {
+			numaInfo.Annotations = make(map[string]string)
+		}
+		numaInfo.Annotations["updatets"] = fmt.Sprint(time.Now().Unix())
```

Comment:
what's the meaning of the key "updates"?

Reply:
updatets means update-timestamp. the value of updatets isn't make sense, but it is in annotations that's make sense. This will trigger numatopo crd force update, it's necessary when a new process start at first time.

Reply:
suggest to change the key to a more stardard one and add the comments as well.

Reply:
ok, I will use name timestamp, remark is use to trigger CR update.